src/iDealAdvancedConnector/XmlSignature/XmlSignature.cs

```diff
@@ -22,13 +22,13 @@ public class XmlSignature
         /// </summary>
         public const string XmlDigSignRSASHA256Namespace = "http://www.w3.org/2001/04/xmldsig-more#rsa-sha256";
 
-        /// <summary>
-        /// Registers the signing algorithm
-        /// </summary>
-        public static void RegisterSignatureAlghorighm()
-        {
-            CryptoConfig.AddAlgorithm(typeof(RSAPKCS1SHA256SignatureDescription), XmlDigSignRSASHA256Namespace);
-        }
+        ///// <summary>
```

Comment:
If it is not needed, we need to remove it.

Reply:
Removed